spec/spec_helper.rb

```diff
@@ -8,14 +8,16 @@
 require "rspec/rails"
 require "sidekiq/testing"
 require 'flipper/adapters/memory'
+require 'mock_redis'
+require 'database_cleaner'
```

Comment:
OOI - were these new includes needed for this PR to work? They should be available via the bundler setup but leave them in if it's breaks without them.

Reply:
Oh! I did need this locally, but noticed that tests were running in GH action without needing this. I'll remove.